content/getting-started/building-a-flutter-app.md

```diff
@@ -21,25 +31,19 @@ You can get started with YAML easily if you have an existing project set up in C
 2. In the **Configuration as code** section on the right sidebar, click **Download configuration**.
 
 {{<notebox>}}
-
 The YAML feature currently has the following **limitations**:
 
 * Exporting configuration from the workflow editor is supported for Flutter-based Android, iOS and web apps.
 * The exported configuration is not identical to the settings in the workflow editor and lacks the configuration for some features, such as **Stop build if tests fail** and publishing to Codemagic Static Pages.
-
 {{</notebox>}}
 
 ## Building with YAML
 
-In order to use `codemagic.yaml` for build configuration in Codemagic, it has to be committed to your repository. The file must be located in the root directory of the repository. When detected in the repository, `codemagic.yaml` is automatically used for configuring builds that are triggered in response to the events defined in the file. Any configuration in the Flutter workflow editor is ignored.
-
-In order to use `codemagic.yaml` for manual builds, you must select the workflow to build from `codemagic.yaml`:
-
-1. Navigate to your app settings and click **Start new build**. 
-3. In the **specify build configuration** popup, select the branch where your `codemagic.yaml` file is located.
-4. If a `codemagic.yaml` file is found in that branch, you can click **Select workflow from codemagic.yaml**.
-5. Then select the workflow to build from the **Select workflow from codemagic.yaml** dropdown.
-6. Finally, click **Start new build** to start the build.
+1. In order to use `codemagic.yaml` for build configuration in Codemagic, commit the file to the root directory of your repository.
+2. Back in app settings in Codemagic, select the **`codemagic.yaml`** tab.
```

Comment:
2 and 3 are optional. Yaml will be searched in Start build modal anyway

Reply:
It's probably more confusing to indicate they are optional, so I think it's fine as is.